.github/ISSUE_TEMPLATE/sprint_planning_retro.md

```diff
@@ -7,6 +7,12 @@ title: Sprint 1.n.0 m/2 - Jan 1 to Jan 12
 
 # Global Sprint Planning
 
+## 3 things that might take us down (owner: @guidoiaquinti)
```

Comment:
Nice 🙇! 

Can we make this as a collective exercise? I don't think there should be a single owner for such a critical list.

```suggestion
## 3 things that might take us down
```